includes/Core/Authentication/Authentication.php

```diff
@@ -364,11 +364,11 @@ function( $user ) {
 					$user['user']['email']   = $profile_data['email'];
 					$user['user']['picture'] = $profile_data['photo'];
 				}
-
-				$user['connectURL']     = esc_url_raw( $this->get_connect_url() );
-				$user['initialVersion'] = $this->initial_version->get();
-				$user['userInputState'] = $this->user_input_state->get();
-				$user['verified']       = $this->verification->has();
+				$user['user']['authenticated'] = $this->is_authenticated();
```

Comment:
The `user` key/property is for data specific to the WP User (with the exception of the email and picture 😄 ). Let's put this on the top-level, as a sibling of `verified` instead.

Reply:
👍